nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadServiceImpl.java

```diff
@@ -105,6 +106,8 @@
 
     private static final String RUN_FUNCTION = "run";
 
+    private static final Pattern JSESSION_ID_REGEX = Pattern.compile(";jsessionid=[^?]*", Pattern.CASE_INSENSITIVE);
```

Comment:
Per my comment about the test, the regexp should take into account other matrix params.

Reply:
Won't that break any query string parameter that includes ```;``` character ?